_episodes/02-filedir.md

```diff
@@ -833,10 +833,48 @@ and we will see it in many other tools as we go on.
 > nevertheless.
 >
 > > ## Solution
-> > The `-l` arguments makes `ls` use a **l**ong listing format, showing not only
-> > the file/directory names but also additional information such as the file size
-> > and the time of its last modification. The `-h` argument makes the file size
-> > "**h**uman readable", i.e. display something like `5.3K` instead of `5369`.
+> > The `-l` argument makes `ls` use a **l**ong listing format, showing not only
+> > the file or directory names but also additional properties of each file or directory. 
```

Comment:
Agree, while "properties" is more technically correct, information is more understanding